install.ps1

```diff
@@ -0,0 +1,59 @@
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+$ProgressPreference = 'SilentlyContinue'
+$release = Invoke-RestMethod -Method Get -Uri "https://api.github.com/repos/ducaale/xh/releases/latest"
+$asset = $release.assets | Where-Object name -like *.zip
```

Comment:
Looking for a zip file seems fragile, would it be better to look for `x86_64-pc-windows` in the name instead?

Reply:
It very much is. Searching by `x86_64-pc-windows` now.  Made a new commit with updated readme and install files.